src/lib/consent.ts

```diff
@@ -0,0 +1,25 @@
+import {
+	type ConsentState,
+	getConsentFor,
+	type VendorName,
+} from '@guardian/libs';
+
+/**
+ * Helper function to check consent for a known TCF vendor that also
+ * takes into account global consent frameworks (USNAT & AUS)
+ * */
+export const hasConsentFor = (
```

Comment:
I'm not sure if this is exactly what you want, but the cmp already does the appropriate checks for each regional framework

https://github.com/guardian/csnx/blob/47c02c76f65a53d638718e5777cec52341a8f398/libs/%40guardian/libs/src/consent-management-platform/getConsentFor.ts

Reply:
I see you found it 👍

Reply:
Easily done!